File: Questão1.py
class Carro:
    """Classe que representa um carro."""

    def __init__(self, marca: str, modelo: str, ano: int) -> None:
        self.marca = marca
        self.modelo = modelo
        self.ano = ano

    def __str__(self) -> str:
        return f"{self.ano} {self.marca} {self.modelo}"

# Instanciando três objetos da classe Carro
carros = [
    Carro("Toyota", "Corolla", 2020),
    Carro("Honda", "Civic", 2019),
    Carro("Ford", "Fiesta", 2021)
]

for carro in carros:
    print(carro)

File: Questão1.java
class Carro {
    private String marca;
    private String modelo;
    private int ano;

    public Carro(String marca, String modelo, int ano) {
        this.marca = marca;
        this.modelo = modelo;
        this.ano = ano;
    }

    @Override
    public String toString() {
        return ano + " " + marca + " " + modelo;
    }

    public static void main(String[] args) {
        Carro[] carros = {
            new Carro("Toyota", "Corolla", 2020),
            new Carro("Honda", "Civic", 2019),
            new Carro("Ford", "Fiesta", 2021)
        };

        for (Carro carro : carros) {
            System.out.println(carro);
        }
    }
}
